android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/action/RoutingAction.kt

```diff
@@ -1,12 +1,16 @@
 package com.badoo.ribs.core.routing.action
 
-import android.os.Bundle
 import com.badoo.ribs.core.Node
-import com.badoo.ribs.core.routing.portal.AncestryInfo
+import com.badoo.ribs.core.builder.BuildContext
 
 interface RoutingAction {
 
-    fun buildNodes(ancestryInfo: AncestryInfo, bundles: List<Bundle?>) : List<Node<*>> =
+    val nbNodesToBuild: Int
```

Comment:
I would try something more elegant, e.g. nodesCount
Also maybe internal ?

Reply:
NodesCount sounds like already existing count. I find current name to be more expressive of idea.

Interfaces are public by design, you can't have internal field, won't compile. Also if client code wants to implement custom RoutingAction, it needs to specify this value.